cola/widgets/main.py

```diff
@@ -692,6 +717,13 @@ def apply_state(self, state):
         diff_numbers = state.get('show_diff_line_numbers', False)
         self.diffeditor.enable_line_numbers(diff_numbers)
 
+        toolbar_data = state.get('toolbar', '[]')
```

Comment:
The default value here must be `[]` instead of the `'[]'` string.. just ran into a traceback when switching into this branch from the state where the settings file does not have this data.